ui/raidboss/data/04-sb/raid/o4s.txt

```diff
@@ -1,211 +1,212 @@
-1150	"Delta Attack 2"	#Blizzard III (Healers & DPS) + Fire III (Tanks & Healers)
-1155	"Flood of Naught"	#1 of 4 Flood of Naught variants
-
-######Neo Phase 5
-1164	"Neverwhere"	#Starts soft enrage
-1173	"Charybdis"	#Everyone to low HP
-1176	"Frenzied Fist x9"	#Small Raid AoEs
-1183	"Frenzied Sphere (1)"	#Big Raid AoE
-1186	"Frenzied Fist x9"	#Small Raid AoEs
-1193	"Frenzied Sphere (2)"	#Big Raid AoE
-1196	"Frenzied Fist x9"	#Small Raid AoEs
-1203	"Frenzied Sphere (3)"	#Big Raid AoE
-1206	"Frenzied Fist x9"	#Small Raid AoEs
-1213	"Frenzied Sphere (4)"	#Big Raid AoE
-1216	"Frenzied Fist x9"	#Small Raid AoEs
-1223	"Frenzied Sphere (5)"	#Big Raid AoE
-1226	"Frenzied Fist x9"	#Small Raid AoEs
-1233	"Frenzied Sphere (6)"	#Big Raid AoE
-1236	"Frenzied Fist x9"	#Small Raid AoEs
-1243	"Frenzied Sphere (7)"	#Big Raid AoE
-1246	"Frenzied Fist x9"	#Small Raid AoEs
-1253	"Frenzied Sphere (8)"	#Big Raid AoE
-1266	"Enrage"	#Long Almagest
+# -ii 23F2 23F8 2400 240C 2410 2413 2414 2415 241B 2421 2422 242F 2431 2432 244C 244D 245E 24DC
+
+# Exfaust
+3.8 "--sync--" sync /:Exdeath:23F2:/ window 3.8,0
+12.2 "Dualcast" sync /:Exdeath:23F4:/ window 12.2,5
+16.8 "--sync--" sync /:Exdeath:23F7:/
+17.3 "Blizzard III" # sync /:Exdeath:23F8:/
+19.9 "Blizzard III" # sync /:Exdeath:23F8:/
+24.8 "Dualcast" sync /:Exdeath:23F4:/
+30.8 "--sync--" sync /:Exdeath:23F9:/
+31.5 "Thunder III" # sync /:Exdeath:23FA:/
+32.6 "Thunder III" # sync /:Exdeath:23FA:/
+37.9 "Dualcast" sync /:Exdeath:23F4:/
+42.5 "--sync--" sync /:Exdeath:23F5:/
+43.1 "Fire III" # sync /:Exdeath:23F6:/
+47.1 "Fire III" # sync /:Exdeath:23F6:/
+53.6 "White Hole" sync /:Exdeath:23FF:/
+64.7 "The Decisive Battle" sync /:Exdeath:2408:/
+66.5 "--untargetable--"
+
+70.6 "--sync--" sync /:Exdeath:240A:/
+71.8 "Collision" sync /:Exdeath:2409:/
+73.8 "Holy" sync /:Exdeath:2403:/
+77.7 "Zombie Breath" sync /:Exdeath:240B:/
+82.4 "Flare" sync /:Exdeath:2401:/
+
+85.2 "--targetable--"
+89.2 "Blizzard III/Fire III/Thunder III" sync /:Exdeath:23F[BCD]:/ window 10,10
+95.4 "Vacuum Wave" sync /:Exdeath:23FE:/
+105.6 "White Hole" sync /:Exdeath:23FF:/
+118.0 "Black Hole" sync /:Exdeath:2406:/
+123.1 "Dualcast" sync /:Exdeath:23F4:/
+127.7 "--sync--" sync /:Exdeath:23F5:/
+128.3 "Fire III" # sync /:Exdeath:23F6:/
+132.2 "Fire III" # sync /:Exdeath:23F6:/
+134.8 "Holy" sync /:Exdeath:240[23]:/
+144.0 "White Hole" sync /:Exdeath:23FF:/
+153.1 "Dualcast" sync /:Exdeath:23F4:/
+159.1 "--sync--" sync /:Exdeath:23F9:/
+159.8 "Thunder III" # sync /:Exdeath:23FA:/
+160.9 "Thunder III" # sync /:Exdeath:23FA:/
+166.1 "Meteor" sync /:Exdeath:2404:/
+179.4 "The Decisive Battle" sync /:Exdeath:2408:/
+181.5 "--untargetable--"
+
+185.3 "--sync--" sync /:Exdeath:240A:/
+186.4 "Collision" sync /:Exdeath:2409:/
+188.7 "Holy" sync /:Exdeath:2403:/
+192.4 "Zombie Breath" sync /:Exdeath:240B:/
+197.1 "Flare" sync /:Exdeath:2401:/
+
+200.0 "--targetable--"
+203.9 "Blizzard III/Fire III/Thunder III" sync /:Exdeath:23F[BCD]:/
```

Comment:
Can you add an English translation to make this `Blizzard/Fire/Thunder III` since this is pretty long?